File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int ans = max(0, gain[0]);
        
        for(int i=1; i<gain.size(); i++) {
            gain[i] += gain[i-1];
            ans = max(ans, gain[i]);
        }
        
        return ans;
    }
};

File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.java
class Solution {
    public int largestAltitude(int[] gain) {
        int ans = Math.max(0, gain[0]);
        
        for(int i=1; i<gain.length; i++) {
            gain[i] += gain[i-1];
            ans = Math.max(ans, gain[i]);
        }
        
        return ans;
    }
}